test/manual/templates/system_api_test.sh

```
@@ -0,0 +1,648 @@
+    response=$(parse_response_safe "$raw")
+    solar_system_id=$(echo "$response" | jq -r '.data[0].solar_system_id // .data[0].id // empty' 2>/dev/null)
+    if [[ -z "$solar_system_id" ]]; then
+      echo "No systems found in the response"
+      return 1
+    fi
+  fi
+  
+  # If using numeric system ID from payload, try that directly
+  if [[ $solar_system_id == "30001660" || $solar_system_id == "30002718" ]]; then
+    # Using the solar system ID directly
+    local payload=$(jq -n '{position_x:123,position_y:456}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)